core/src/main/java/org/apache/iceberg/util/ThreadPools.java

```diff
@@ -61,6 +61,15 @@ public static ExecutorService getWorkerPool() {
     return WORKER_POOL;
   }
 
+  /**
+   * Return parallelism for "worker" thread-pool.
+   * In default, we submit 2 tasks per worker at a time.
+   * @return the parallelism of the worker pool.
+   */
+  public static int getPoolParallelism() {
```

Comment:
I am wondering if it is better to define a constant as 
```
public static final int DEFAULT_WORKER_THREAD_POOL_PARALLELISM = WORKER_THREAD_POOL_SIZE * 2;
```

Reply:
LGTM, I will change that as a constant.